frontends/web/src/api/exchanges.ts

```diff
@@ -33,6 +33,14 @@ export const getExchangesByRegion = (code: string) => {
   };
 };
 
+
+export const getSellExchangesByRegion = () => {
+  return (): Promise<ExchangeRegionList> => {
+    return Promise.resolve({ regions: [{ code: 'CH', isMoonpayEnabled: true, isPocketEnabled: false }] });
+  };
+};
```

Comment:
just fyi: We used the "curry" function style in other places if the api fn takes an argument, but then started just doing `useLoad(() => apiCall(withArg))` instead of `useLoad(apiCall(withArg))` (which requires the currying). 
I think we should just use `useLoad(() => apiCall(withArg))` when possible and not curry. wdyt?

Reply:
Yeah I agree - basically immediately return the apiGet, instead of returning a promise, then inside of that promise we return the apiGet, right?

So, do this:

```ts
export const getExchangeBuySupported = (code: AccountCode): Promise<SupportedExchanges> => {
   return apiGet(`exchange/buy-supported/${code}`);
};
```

Instead of the current way:
```ts
export const getExchangeBuySupported = (code: AccountCode) => {
  return (): Promise<SupportedExchanges> => {
    return apiGet(`exchange/buy-supported/${code}`);
  };
};
```

is my understanding right?

Reply:
Yes exactly, and I think we started doing it like this with `useLoad(() => getVersionInfo(deviceID))` in a few places.

Reply:
Yup - changes made. Looks cleaner without currying.